projects/EZZY-game-BSC-token/index.js

```diff
@@ -0,0 +1,14 @@
+const { staking } = require("../helper/staking")
+
+const contract = "0xc056124727540Fc26C3AF44d2b6BA4FbF59C6909"
```

Comment:
this contract belongs to pancake swap,c ant be included in your tvl or staking

Reply:
But we have only liquidity pool, how can we count token's TVL with it? I tried to use pool2 adapter, but i get these error
<img width="640" alt="image" src="https://user-images.githubusercontent.com/45100335/236175026-1ef91dc8-254a-44c3-8064-cab10e38cabe.png">

Reply:
okay, this pool doesnt belong to you.

Do your users stake these LPs in smart contracts controlled by your project?

https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
Our user doesn't stake LP tokens, but our team locked LP tokens for 3 years on umicrypt smartcontact. can we somehow to show on defilama info about liqidity pool or LP tokens locked?

Reply:
sorry, we currently do not support that